manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -132,10 +119,7 @@ spec:
     rules:
     - alert: etcdHighNumberOfLeaderChanges
       annotations:
-        description: 'etcd cluster "{{ $labels.job }}": {{ $value }} leader changes
-          within the last 15 minutes. Frequent elections may be a sign of insufficient
-          resources, high network latency, or disruptions by other components and
-          should be investigated.'
+        description: 'etcd cluster "{{ $labels.job }}": {{ $value }} leader changes within the last 15 minutes. Frequent elections may be a sign of insufficient resources, high network latency, or disruptions by other components and should be investigated.'
```

Comment:
Something wrong with your formating or why is this change? 🤔

Reply:
https://github.com/openshift/cluster-etcd-operator/pull/632#pullrequestreview-713546641